### PR TITLE
Bayesian Linear regression vs Plugin approximation (linreg_post_pred_plot.ipynb)

### PR DESCRIPTION
**Pull request by AnandShegde**
## Description
Jaxified and latexified the figures in linreg_post_pred_plot.ipynb.

### Figure Number
3.1

### Figures
[gist](https://gist.github.com/AnandShegde/f2886d763ba3669146828cbae7fea23f)
|Before PR|After PR|
| :-: |:-:|
| before latexifying |After PR
|

### Issue 
#37 

## Steps

- [x] Step1 Used Jax instead of Numpy wherever possible.
- [x] Step2 Latexified the images.

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

## Potential problems/Important remarks
This is a copy of the pull request that I created earlier #47. I tried to undo the to other notebook in the branch and had messed up with the pull request too. [Here](https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/47/) are the remarks that you had given. I have correced them now. Please take some time and review the notebook. 
Thank you.

### CODE REVIEW

**Comment by AnandShegde** [comment]
I have updated the legend size as we had discussed. Please take some time and review the pull request.

**Comment by karm-patel** [comment]
1. For xlabel and ylabel, always refer to textbook caption and related text, did they use captical `X` & `Y` somewhere? I think it should be lowercase

File: notebooks/book2/03/linreg_post_pred_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,352 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/03/linreg_post_pred_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    NFuncSamples = 10  # Number of sample coefficients to draw and use for prediction
</pre><p>use lowercase in variable names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (karm-patel) [inline notebooks/book2/03/linreg_post_pred_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.    def shp(x):
</pre><p><code>shp</code> is not meaningful name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: notebooks/book2/03/linreg_post_pred_plot.ipynb [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
```
Comment (karm-patel) [inline notebooks/book2/03/linreg_post_pred_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.    plt.show()
</pre><p>get rid of this warnings </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by karm-patel** [comment]
Update new laexified figure in PR template

**Comment by karm-patel** [comment]
Go ahead for PR

**Pull request merged**